src/overview/tabs/Overview/utils/timeRangeOptions.ts

```diff
@@ -34,7 +34,7 @@ export const TimeRangeOptionsDictionary: {
   All: TimeRangeOptionsProperties;
 } = {
   All: {
-    bucket: { day: 1 },
+    bucket: { day: 4 },
```

Comment:
please add constant with meaningful name

Reply:
I think it's understood in context, I could create const like FOUR_DAYS but not sure it's needed here?

Reply:
prefer the constant please